frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/CheckReAuthUserHandler.java

```diff
@@ -30,11 +30,29 @@
 public class CheckReAuthUserHandler extends BaseFrontendHandler<CheckReauthUserRequest>
         implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
 
+    // INIT PHASE
     private static final Logger LOG = LogManager.getLogger(CheckReAuthUserHandler.class);
+    private static RedisConnectionService redis;
 
-    private final AuditService auditService;
-    private final CodeStorageService codeStorageService;
+    static {
+        if (System.getProperty("TEST") == null) {
+            redis = new RedisConnectionService(ConfigurationService.getInstance());
+        }
+    }
+
+    // FUNCTION INIT
+    public CheckReAuthUserHandler() {
+        this(ConfigurationService.getInstance(), redis);
```

Comment:
If the connection breaks; is there anything here that will re-connect - or will the lambda simply have a broken connection?

Reply:
At the moment the first call that uses the broken connection will fail and cause an RTE to be thrown which, in turn, will cause the lambda to restart.  If we see improvement from this change then the next step will be to add in better handling of errors on the redis client including broken connections.